packages/starrating/src/react/__stories__/index.story.tsx

```diff
@@ -16,14 +16,20 @@ const rounding = storiesOf(
 )
 
 VALUES.forEach(value => {
-  interactive.add(`${value}`, _ => (
+  interactive.add(`${value}`, () => (
     <StarRating onChange={action('value changed')} value={value} />
   ))
-  nonInteractive.add(`${value}`, _ => <StarRating value={value} />)
+  nonInteractive.add(`${value}`, () => <StarRating value={value} />)
 })
 
 ROUNDING_VALUES.forEach(value => {
-  rounding.add(`${value}`, _ => (
-    <StarRating onChange={action('value changed')} value={value} />
+  rounding.add(`${value}`, () => (
+    <StarRating
+      onChange={(
+        val: React.Key,
+        evt: React.ChangeEvent<Element> | React.MouseEvent<Element, MouseEvent>
```

Comment:
```suggestion
        _val: React.Key,
        _evt: React.ChangeEvent<Element> | React.MouseEvent<Element, MouseEvent>
```

It's too bad that tsc apparently requires typing this.  Storybook's `action` was always so nice and terse.